img/beam-robotics/codingrobot.svg

```diff
@@ -0,0 +1,67 @@
+<path d="M163.644 50.7919C169.73 50.7919 174.681 45.8407 174.681 39.7552C174.681 33.6696 169.73 28.7184 163.644 28.7184C157.558 28.7184 152.607 33.6696 152.607 39.7552C152.607 45.8407 157.558 50.7919 163.644 50.7919ZM163.644 33.6291C167.022 33.6291 169.77 36.377 169.77 39.7549C169.77 43.1327 167.022 45.8806 163.644 45.8806C160.267 45.8806 157.518 43.1327 157.518 39.7549C157.518 36.377 160.267 33.6291 163.644 33.6291Z" fill="#4D4D4D"/>
+<path d="M165.806 80.2987C167.162 80.2987 168.261 79.1993 168.261 77.8431C168.261 76.4869 167.162 75.3875 165.806 75.3875C164.449 75.3875 163.35 76.4869 163.35 77.8431C163.35 79.1993 164.449 80.2987 165.806 80.2987Z" fill="#4D4D4D"/>
+<path d="M131.194 80.2987C132.551 80.2987 133.65 79.1993 133.65 77.8431C133.65 76.4869 132.551 75.3875 131.194 75.3875C129.838 75.3875 128.739 76.4869 128.739 77.8431C128.739 79.1993 129.838 80.2987 131.194 80.2987Z" fill="#4D4D4D"/>
+<path d="M148.502 8.50762C150.851 8.50762 152.756 6.60313 152.756 4.25381C152.756 1.9045 150.851 0 148.502 0C146.152 0 144.248 1.9045 144.248 4.25381C144.248 6.60313 146.152 8.50762 148.502 8.50762Z" fill="#F1A25C"/>
+<path d="M134.871 84.376H162.129C162.129 87.5552 161.038 90.4822 159.215 92.8004C156.717 95.9698 152.847 98.0062 148.499 98.0062C140.971 98.0062 134.871 91.9031 134.871 84.376Z" fill="#CECDCE"/>
+<path d="M134.871 86.668H159.644C159.644 89.8472 158.553 92.7742 156.73 95.0924C154.412 96.9161 151.488 98.0062 148.305 98.0062C141.56 98.0062 135.958 93.105 134.871 86.668Z" fill="#F1A25C"/>
+</g>
+<path d="M34.8854 146.521C49.1874 157.431 58.1618 158.163 73.9757 156.517C83.1764 155.56 91.9772 151.581 99.3865 159.789C105.031 166.042 108.788 164.95 115.937 164.206C124.184 163.348 122.304 156.185 121.406 147.563" stroke="#5EDA12" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M120.351 135.995C119.898 141.849 125.583 155.746 106.919 162.075C89.8791 167.853 91.6798 154.002 85.4202 150.984C79.0084 147.893 72.6993 141.82 62.013 141.869C56.8675 141.892 54.0127 140.277 48.6481 140.776C45.0623 141.109 39.7664 143.389 36.0403 144.652" stroke="#007A58" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M38 145C51.2763 151.539 62.3673 166.375 76.3718 170.768C82.5803 172.716 80.591 161.867 82.3291 157.266C84.5745 151.322 94.0709 149.523 98.6239 148.298C106.596 146.154 112.232 146.855 120 146.855" stroke="#FF0000" stroke-width="3" stroke-linecap="round" stroke-linejoin="round"/>
+<defs>
+<clipPath id="clip0">
+<rect width="53" height="53" fill="white" transform="translate(0 94)"/>
+</clipPath>
+<clipPath id="clip1">
+<rect width="147" height="147" fill="white" transform="translate(75)"/>
+</clipPath>
+</defs>
+</svg>
```

Comment:
rename the dancingrobot.svg & m_bot.svg =>mBot.svg

Reply:
I made m_bot.svg to mBot.svg, but since dancingrobot.svg is a different file I just renamed it to dancing-robot.svg similar to coding-robot.svg